azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -158,19 +161,45 @@ def call_activity(self, name: str, input_: Optional[Any] = None) -> TaskBase:
         Task
             A Durable Task that completes when the called activity function completes or fails.
         """
+        if(isinstance(name, Callable)):
+            if (isinstance(name, FunctionBuilder)):
+                try:
+                    if (isinstance(name._function._trigger, ActivityTrigger)):
+                        name = name._function._name
+                    else:
+                        error_message = "Received function with Trigger-type `"\
+                            + name._function._trigger.type\
+                            + "` but expected `ActivityTrigger`. Ensure your "\
+                            "function is annotated with the `activity_trigger` decorator "\
+                            "or directly pass in the name of the activity as a string."
+                        raise Exception(error_message)
+                except AttributeError as e:
+                    e.message = e.message + "Attribute couldn't be found or missing. "\
+                        "Please check if your code acts in error, "\
+                        "or file a bug in Durable Python repository: "\
+                        "https://github.com/Azure/azure-functions-durable-python.git."
```

Comment:
Please consider this phrasing:

```suggestion
                    e.message = "Durable Functions SDK internal error: an expected attribute is missing from the `FunctionBuilder` object in the Python V2 programming model. Please report this bug in the Durable Functions Python SDK repo: https://github.com/Azure/azure-functions-durable-python.git.\n Error trace: " + e.message 
```